library/auth/src/main/java/com/epmedu/animeal/auth/AuthAPIImpl.kt

```diff
@@ -44,58 +38,33 @@ internal class AuthAPIImpl(
             userPoolTokensResult.type == AuthSessionResult.Type.SUCCESS &&
             userSubResult.type == AuthSessionResult.Type.SUCCESS
 
-    override suspend fun getCurrentUserId(): String = suspendCancellableCoroutine {
-        Amplify.Auth.getCurrentUser(
-            { user -> resume(user.username) },
-            { authException ->
-                if (isRefreshTokenHasExpiredException(authException)) {
-                    handleRefreshTokenExpiration()
-                } else {
-                    resumeWithException(authException)
-                }
-            }
-        )
-    }
+    override suspend fun getCurrentUserId() = Amplify.Auth.getCurrentUser().userId

```

Comment:
We must return username instead of user id here for a reason. Don't change this to avoid breaking the app.